### PR TITLE
add mssql and nodejs

### PR DESCRIPTION
**Pull request by KhanbalaRashidov**
add mssql and nodejs script

### CODE REVIEW

File: mssql/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+echo "load env.."
+. env.sh
+bash ../remove.sh "$CONTAINER_NAME"
+sudo docker run --detach \
+    --name "$CONTAINER_NAME" \
+    --env=MSSQL_SA_PASSWORD="$MSSQL_SA_PASSWORD" \
+    --env=MSSQL_PID="$MSSQL_PID" \
```
Comment (rad1k4l) [inline mssql/deploy.sh:8]: why we set sql server pid(process id)  manual ? it must be provided by OS. Or what `MSSQL_PID` means ?

File: mssql/env.example.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+CONTAINER_NAME="mssql"
+MSSQL_SA_PASSWORD="x3h3Ays7dNdUGkYVKyDEMkHVEUeFyDr8"
+MSSQL_PID="Enterprise"
```
Comment (rad1k4l) [inline mssql/env.example.sh:3]: Add readme to `mssql` folder, and explain which values support this field.

File: nodejs/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+. env.sh
+bash ../remove.sh "$CONTAINER_NAME"
+sudo docker run --detach \
+    --name "$CONTAINER_NAME" \
+    --publish 3000:3000 \
+    --env= "$Node_ENV" \
+    --env="$APP" \
+    --network="$NETWORK" \
+    --volume "$WEB_PATH":/var/www/example.com/api \
+    node:"$VERSION"
```
Comment (rad1k4l) [inline nodejs/deploy.sh]: add empty line

File: nodejs/env.example.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+CONTAINER_NAME="nodejs-app"
+WEB_PATH="/var/docker/nodejs/data/api"
+NODE_ENV="development" 
+APP="index.js"
+NETWORK="main"
+VERSION="14.19.3"
```
Comment (rad1k4l) [inline nodejs/env.example.sh]: add empty line

File: nodejs/deploy.sh [hunk 5]
```diff
@@ -0,0 +1,11 @@
+echo "load env.."
+. env.sh
+bash ../remove.sh "$CONTAINER_NAME"
+sudo docker run --detach \
+    --name "$CONTAINER_NAME" \
+    --publish 3000:3000 \
+    --env= "$Node_ENV" \
+    --env="$APP" \
```
Comment (rad1k4l) [inline nodejs/deploy.sh]: env supports KEY=VALUE type

File: nodejs/deploy.sh [hunk 6]
```diff
@@ -0,0 +1,11 @@
+echo "load env.."
+. env.sh
+bash ../remove.sh "$CONTAINER_NAME"
+sudo docker run --detach \
+    --name "$CONTAINER_NAME" \
+    --publish 3000:3000 \
+    --env= "$Node_ENV" \
```
Comment (rad1k4l) [inline nodejs/deploy.sh]: env supports key=value type syntax

**Review by rad1k4l — CHANGES_REQUESTED** [review]

File: mssql/env.example.sh (hunk 2 above)
Comment (KhanRashidov) [reply mssql/env.example.sh:3]: you can reach what it is for from this link https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-configure-environment-variables?view=sql-server-ver16

File: mssql/deploy.sh (hunk 1 above)
Comment (KhanbalaRashidov) [reply mssql/deploy.sh:8]: you can reach what it is for from this link https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-configure-environment-variables?view=sql-server-ver16

Comment (KhanbalaRashidov) [reply mssql/deploy.sh:8]: The pid you see here shows the edition or product key, not the process id.

Comment (rad1k4l) [reply mssql/deploy.sh:8]: resolved

File: nodejs/env.example.sh (hunk 4 above)
Comment (KhanbalaRashidov) [reply nodejs/env.example.sh]: where?

**Comment by KhanbalaRashidov** [comment]
Added README.md

File: nodejs/env.example.sh (hunk 4 above)
Comment (rad1k4l) [reply nodejs/env.example.sh]: Add to last line

**Review by rad1k4l — CHANGES_REQUESTED** [review]
Resolve conversations

**Review by rad1k4l — APPROVED** [review]

**Comment by rad1k4l** [comment]
Fix conversation. ->  env supports KEY=VALUE type

**Comment by KhanRashidov** [comment]
> Fix conversation. -> env supports KEY=VALUE type

already so

**Review by rad1k4l — APPROVED** [review]

**Pull request merged**